etc/profile-m-z/virtualbox.profile

```diff
@@ -14,19 +14,34 @@ noblacklist /usr/lib/virtualbox
 noblacklist /usr/lib64/virtualbox
 
 include disable-common.inc
+include disable-devel.inc
 include disable-exec.inc
+include disable-interpreters.inc
 include disable-passwdmgr.inc
 include disable-programs.inc
+include disable-xdg.inc
 
 mkdir ${HOME}/.config/VirtualBox
 mkdir ${HOME}/VirtualBox VMs
 whitelist ${HOME}/.config/VirtualBox
 whitelist ${HOME}/VirtualBox VMs
 whitelist ${DOWNLOADS}
 include whitelist-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
 include whitelist-var-common.inc
 
-caps.keep net_raw,sys_admin,sys_nice
```

Comment:
Sure this does not break? On which distros have you tested it?

Reply:
I have tested on manjaro(KDE). Kernel version: 5.7.9. you can even use `caps.drop all` . Do you have any profile testing community or group?

Reply:
`caps.keep net_raw,sys_admin,sys_nice` 
it's needed to be in comment